administrator/components/com_templates/config.xml

```diff
@@ -16,6 +16,17 @@
 			<option value="1">JENABLED</option>
 		</field>
 
```

Comment:
Isn't this switch only possible if the preview is allowed? If yes: you could make a dpendecy with showon="template_positions_display:1"

Reply:
Actually, I've kept them independent so the ?pm (place module view) does not need ?tp (template positions to be enabled). I did this because many admins would prefer to have Add Modules enabled while only developers would know and require the need for Template Positions and also because Place Modules will be enabled by default upon installation. 

Should I keep Place Modules dependent on template_positions?

Reply:
no, good argument